src/inc/regdisp.h

```diff
@@ -53,6 +53,11 @@ struct REGDISPLAY_BASE {
 #endif // !_TARGET_X86_ && !_TARGET_ARM
 };
 
+inline PCODE GetControlPC(REGDISPLAY_BASE *pRD) {
```

Comment:
It seems it would be nice to unify also the GetRegdisplaySP and SetRegdisplaySP. I wonder if we can even unify the ControlPC and SP in the REGDISPLAY_BASE to be of size_t type for all targets.